packages/riverpod_generator/lib/src/models.dart

```diff
@@ -30,6 +36,13 @@ extension CaseChangeExtension on String {
     );
   }
 
+  String get upperFirst {
```

Comment:
There's a `string.titled` extension already.

Reply:
I removed it in [refactor: Use titled instead of upperFirst](https://github.com/rrousselGit/riverpod/pull/3348/commits/1a6aee1d676bcb8ed496b0f2c5019f3ad00ec97d)